src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -73,25 +77,44 @@ func sessionVals(ip, version, clientId, sc string) string {
 	// Custom variable for the Lantern version
 	vals.Add("cd1", version)
 
+	// Custom dimension for the hash of the executable
+	hash := getExecutableHash()
+	vals.Add("cd2", hash)
+
 	// This forces the recording of the session duration. It must be either
 	// "start" or "end". See:
 	// https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters
 	vals.Add("sc", sc)
 	return vals.Encode()
 }
 
-func endSession(ip string, version string, proxyAddrFN eventual.Getter, clientId string) {
-	args := sessionVals(ip, version, clientId, "end")
+// GetExecutableHash returns the hash of the currently running executable.
+// If there's an error getting the hash, this returns
+func getExecutableHash() string {
+	if lanternPath, err := osext.Executable(); err != nil {
```

Comment:
Seems `osext` doesn't support Android. It doesn't halt the program anyway, but would be good to make it explicit.